cvat-core/src/server-proxy.js

```diff
@@ -641,6 +641,8 @@
                         },
                     );
                 } catch (errorData) {
+                    errorData.message = "";
+                    errorData.response.data = String.fromCharCode.apply(null, new Uint8Array(errorData.response.data));
```

Comment:
Please, try something like this:

```js
throw generateError({
    ...errorData,
    message: '',
    response: {...errorData.response, data: String.... },
})
```

For strings use ``'`` instead of ``"``
```